activerecord/lib/active_record/railtie.rb

```diff
@@ -235,5 +235,13 @@ class Railtie < Rails::Railtie # :nodoc:
         end
       end
     end
+
+    initializer "active_record.set_filter_attributes" do
+      config.after_initialize do
+        ActiveSupport.on_load(:active_record) do
+          ActiveRecord::Base.filter_attributes = Rails.application.config.filter_parameters
```

Comment:
These hooks are run in the context of `ActiveRecord::Base`, and it's idiomatic to use `self` instead of referring to the constant explicitly, e.g:

https://github.com/rails/rails/blob/e6ba30efbf107954db2707410d78d77680d69995/activerecord/lib/active_record/railtie.rb#L135-L137